web/src/actions/auth.js

```diff
@@ -2,8 +2,6 @@ import axios from '../util/api';
 
 import { fetchApd } from './apd';
 
-const API_URL = process.env.API_URL || 'http://localhost:8000';
```

Comment:
We can rely on the base URL when we first setup the API library, so no need to put it here.